### DISCUSSION TITLE
carla runs stuck

### DISCUSSION THREAD

**Discussion by Willis814** [carla-simulator/carla#6937] [Q&A]
I would like to know what are the minimum configuration requirements for carla? My computer has NVIDIA 409032G graphics card and Intel 12th Gen Intel(R) Core(TM) i7-12700F 2.10 GHz. Why does it freeze when I run carla (I configured it on Windows 11)

**Comment by Jinviz** [comment]
I don't think it's a problem at all. Why don't you reinstall it. I tried it on gtx 1650s as well, but there was no problem.
